pkg/community_images/list.go

```diff
@@ -47,23 +48,27 @@ func ListImages(configFlags *genericclioptions.ConfigFlags, imageNameCh chan str
 	if err != nil {
 		return nil, errors.Wrap(err, "failed to create clientset")
 	}
-
-	namespaces, err := clientset.CoreV1().Namespaces().List(context.Background(), metav1.ListOptions{})
-	if err != nil {
-		return nil, errors.Wrap(err, "failed to list namespaces")
+	var namespaces []string
+	if len(includeNs) > 0 {
```

Comment:
prevent unnecessary list namespaces api request when include_ns option is specified.